livekit-plugins/livekit-plugins-elevenlabs/livekit/plugins/elevenlabs/tts.py

```diff
@@ -156,6 +156,7 @@ def __init__(
     ) -> None:
         super().__init__()
         self._text, self._opts, self._session = text, opts, session
+        self._mp3_decoder = utils.codecs.Mp3StreamDecoder()
```

Comment:
```suggestion
        self._mp3_decoder = utils.codecs.Mp3StreamDecoder()
```
not: the mp3 decoder isn't available without using livekit-agents[codec]. Can you initialize it inside the condition `if encoding == "mp3":`